sequencer/src/block.rs

```diff
@@ -242,3 +256,120 @@ impl Block {
         &self.l1_block
     }
 }
+
+#[cfg(test)]
+mod reference {
```

Comment:
I think it would be nice for consumers if we commit json files with the test data to this repo.